File: app/src/main/java/net/veldor/flibustaloader/ui/LoaderActivity.java
package net.veldor.flibustaloader.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

public class LoaderActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/net/veldor/flibustaloader/ui/LoaderActivity.kt
package net.veldor.flibustaloader.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.content.Intent

class LoaderActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val intent = Intent(this, MainActivity::class.java)
        startActivity(intent)
        finish()
    }
}